open_source_licenses.txt

```diff
@@ -1,6 +1,6 @@
 open_source_licenses.txt
 
-Wavefront by VMware Internal Reporter for Java 0.9.0 GA
+Wavefront Internal Reporter for Java 0.9.1 GA

```

Comment:
Should this be: "Wavefront by VMware" Internal Reporter for Java ?

Reply:
fixed.